<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAffairDialog</class>
 <widget class="QDialog" name="AddAffairDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_startTime">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>startTime</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_endTime">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>endTime</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_content">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>content</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_urgency">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>urgency</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_category">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>category</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_content">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_urgency">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_category">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="edit_startTime">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>194</width>
     <height>24</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d H:mm</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="edit_endTime">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>194</width>
     <height>24</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_id">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
